cmd/utils/nodecmd/nodeflags.go

```diff
@@ -283,6 +283,7 @@ var KENFlags = []cli.Flag{
 	altsrc.NewUint64Flag(utils.TxResendIntervalFlag),
 	altsrc.NewIntFlag(utils.TxResendCountFlag),
 	altsrc.NewBoolFlag(utils.TxResendUseLegacyFlag),
+	altsrc.NewBoolFlag(utils.UnsafeDebugDisableFlag),
```

Comment:
I think the flag is supposed to be `CommonRPCFlags"

Reply:
Thanks, resolved [b83e083](https://github.com/klaytn/klaytn/pull/1746/commits/b83e0832b21a60dfb66c7f1b9793825311825111)